src/renovate.ts

```diff
@@ -31,6 +32,13 @@ class Renovate {
       );
     }
 
+    if (this.input.mountDockerSocket()) {
+      dockerArguments.push(
+        '--volume /var/run/docker.sock:/var/run/docker.sock',
+        `--user 0:${this.getDockerGroupId()}`
```

Comment:
no, use add-group option. running as root will cause a lot of issues.

Reply:
mmh can you develop on that add-group option thing? How would you do that?

Reply:
would `--user 1000:${this.getDockerGroupId()}` fit since it's the user you create in the Docker image?

Reply:
@viceice not being root [makes renovate's own `install-tool golang 1.20.4` fail](https://github.com/mtardy/tetragon/actions/runs/5072065088/jobs/9109274711#step:5:8240), then I cannot proceed because Go is not installed. This is a run with `--user 1000:${this.getDockerGroupId()}`.

Reply:
sorry, the param is `--group-add`

https://docs.docker.com/engine/reference/commandline/run/

Reply:
```suggestion
        `--group-add ${this.getDockerGroupId()}`
```

Reply:
Just tried it works great!! thanks! Will update the PR.